### PR TITLE
Create 2024-07-17.md

### PR DESCRIPTION
**Pull request by camronmartinez**

### CODE REVIEW

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:1]: Could we mention the API and link to the documentation here?

**Review by joeabraun — CHANGES_REQUESTED** [review]

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, see https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:1]:
```suggestion
Modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, visit the [documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current). 
```

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, see https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current. 
+*	The list of ACA specific error codes previously listed in the description for the HTTP 200 Successful Response schema were removed (errors are not applicable for HTTP 200 Successful Responses).
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:2]:
```suggestion
*	The list of ACA specific error codes in the description for the HTTP 200 Successful Response schema were removed. Errors are not applicable for HTTP 200 Successful Responses.
```

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+*	HTTP 404 error was removed from OAS document as unnecessary (this error code will not be generated for the API).
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:3]:
```suggestion
*	HTTP 404 error was removed from the OAS. This error code will not be generated for the API.
```

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, see https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current. 
+*	The list of ACA specific error codes previously listed in the description for the HTTP 200 Successful Response schema were removed (errors are not applicable for HTTP 200 Successful Responses).
+*	HTTP 404 error was removed from OAS document as unnecessary (this error code will not be generated for the API). 
+* HTTP 422 error was modified in the application to return the error as shown in the OAS documentation (includes “V1000 – Data Error/Invalid Request”).
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:4]:
```suggestion
* HTTP 422 error was modified to return the error as shown in the OAS documentation (which includes “V1000 – Data Error/Invalid Request”). 
```

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, see https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current. 
+*	The list of ACA specific error codes previously listed in the description for the HTTP 200 Successful Response schema were removed (errors are not applicable for HTTP 200 Successful Responses).
+*	HTTP 404 error was removed from OAS document as unnecessary (this error code will not be generated for the API). 
+* HTTP 422 error was modified in the application to return the error as shown in the OAS documentation (includes “V1000 – Data Error/Invalid Request”). 
+* Example pictures were modified to match the OAS schema for the following HTTP errors:  429, 500, 502, and 503.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:5]:
```suggestion
* Example pictures were changed to match the OAS schema for the following HTTP errors: 429, 500, 502, and 503.  
```

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, see https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current. 
+*	The list of ACA specific error codes previously listed in the description for the HTTP 200 Successful Response schema were removed (errors are not applicable for HTTP 200 Successful Responses).
+*	HTTP 404 error was removed from OAS document as unnecessary (this error code will not be generated for the API). 
+* HTTP 422 error was modified in the application to return the error as shown in the OAS documentation (includes “V1000 – Data Error/Invalid Request”). 
+* Example pictures were modified to match the OAS schema for the following HTTP errors:  429, 500, 502, and 503.  
+* HTTP 413 error was added to the OAS document (not previously included).
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:6]:
```suggestion
* HTTP 413 error was added to the OAS. It was not previously included.

For more information, review the [documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current).
```

Comment (joeabraun) [reply content/verification/veteran-verification/release-notes/2024-08-10.md:6]: @camronmartinez what does this error message mean?

File: content/verification/veteran-verification/release-notes/2024-08-10.md (hunk 2 above)
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:1]:
```suggestion
These modifications were made to the OAS documentation for the Veteran Service History and Eligibility API:
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
@camronmartinez requested some changes.

File: content/verification/veteran-verification/release-notes/2024-08-10.md (hunk 7 above)
Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-08-10.md:6]: 413 is a payload too large error. Indicates that the request entity was larger than limits defined by the server.

**Review by joeabraun — CHANGES_REQUESTED** [review]
A couple of smaller changes!

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+Modifications were made to the OAS documentation for the Veteran Service History and Eligibility API. For more information, visit the [documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current). 
+*	The list of ACA specific error codes in the description for the HTTP 200 Successful Response schema were removed. Errors are not applicable for HTTP 200 Successful Responses.
```
[the remark this answers is not in the document]
Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-08-10.md:2]: (Affordable Care Act) There is an ACA elibility endpoint on our API. The above is in reference to specific errors codes for that endpoint in the description for the HTTP 200 successful schema were removed.  
image

File: content/verification/veteran-verification/release-notes/2024-08-10.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Modifications were made to the OAS documentation for the Veteran Service History and Eligibility API.
+*	The list of ACA specific error codes in the description for the HTTP 200 Successful Response schema were removed. Errors are not applicable for HTTP 200 Successful Responses.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-08-10.md:2]:
```suggestion
*	The list of Affordable Care Act (ACA) specific error codes were removed from the description for the HTTP 200 Successful Response schema. Errors are not applicable for HTTP 200 Successful Responses.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

**Review by joeabraun — APPROVED** [review]
Looks good to me!

**Pull request merged**